### DISCUSSION TITLE
Datenpunkt Stil

### DISCUSSION THREAD

**Discussion by Mephisto090** [mcuiobroker/docuJarvis#47] [Q&A]
Hej,

ich vergleiche hier zwei DP als String miteinander (Siehe Bilder). Irgendwie wird die richtige Version aber nicht grün. Früher ging es :(
Weißt du was mein Fehler ist?

Jarvis Settings:
![image](https://github.com/mcuiobroker/docuJarvis/assets/85995176/59f9bb3d-8288-41bf-b699-a92add2502eb)

Tasmota Version:
![image](https://github.com/mcuiobroker/docuJarvis/assets/85995176/c69c1675-b3c1-4363-9fae-1d109582ce62)

Version am Schalter Arbeitszimmer:
![image](https://github.com/mcuiobroker/docuJarvis/assets/85995176/ff4e376b-45a9-4350-aa9d-bbf998649273)

Version am Schalter Treppe:
![image](https://github.com/mcuiobroker/docuJarvis/assets/85995176/dbd9fa9b-a005-4b06-8479-9e70c233c91d)

Jarvis Ausgabe:
![image](https://github.com/mcuiobroker/docuJarvis/assets/85995176/56b03d1e-1220-4023-877e-b776670dd16f)

VG

**Comment by mcuiobroker** [comment]
Nimm mal die Hochkomma bei val raus

**Reply by Mephisto090** [reply]
leider nein :(

**Reply by Mephisto090** [reply]
```
{
  "'{javascript.0.Tasmota.tasmota.deviceVersion}'={val}": {
    "color": "green"
  },
  "default": {
    "color": "red"
  }
}
```
![image](https://github.com/mcuiobroker/docuJarvis/assets/85995176/f14c2ac4-97ee-4f5d-9ff6-840a4e19f0ee)

**Reply by mcuiobroker** [reply]
Welche jarvis Version nur v3.1.9-beta.2?

**Reply by Mephisto090** [reply]
v3.2.0-alpha.26

**Reply by mcuiobroker** [reply]
Da müsste es schon ein Bug Issue innjarvis geben?

**Reply by Mephisto090** [reply]
Kann sein, weiß ich nicht. hast ein link?

**Comment by mcuiobroker** [comment]
https://github.com/Zefau/ioBroker.jarvis/issues/2487

https://github.com/Zefau/ioBroker.jarvis/issues/2490#issuecomment-1870696375
